api/grpc/server_integration_test.go

```diff
@@ -0,0 +1,138 @@
+// build +integration
+
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package grpc_test
+
+import (
+	"context"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/test/bufconn"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+
+	api "github.com/optakt/flow-dps/api/grpc"
+	"github.com/optakt/flow-dps/models/dps/mock"
+)
+
+const bufSize = 1024 * 1024
+
+var lis *bufconn.Listener
+
+var (
+	wantHeight uint64 = 128
+	lastHeight uint64 = 256
+	testValue         = []byte(`testValue`)
+	testKey           = []byte(`testKey`)
+	lastCommit        = flow.StateCommitment{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 1, 2}
+
+	testValues = []ledger.Value{ledger.Value(`testValue`)}
+)
+
+func TestMain(m *testing.M) {
+	mock := mock.NewState()
```

Comment:
Same question, why are we suddenly using testify mocks with no strict typing on the programming?